login_reg_app/models.py

```diff
@@ -106,6 +106,8 @@ class User(models.Model):
 	# max_length to 255, since it could get cut off and therefore not compare the same hash
     password = models.CharField(max_length=255)
 
+    isWorker = models.BooleanField()
```

Comment:
this is the boolean logic you were referring to. This was better than going with a separate class